### ISSUE TITLE
Annual memory playback

### ISSUE THREAD

**Issue by si** [tootsuite/mastodon#8749]
Similar to the Facebook memories, playback content from previous years to reflect on your past. Probably best as a filter rather than inline content that disrupts your timeline.

**Comment by MaciekBaron** [comment]
This sounds more like a job for a bot, not something that should be in core Mastodon.

**Comment by asos-simonj** [comment]
Good idea. It would be nice to have a bot you could follow on each instance (like @lastyear@mastodon.instance) to playback your content from previous years as a private toot or DM.

**Comment by si** [comment]
Good idea. It would be nice to have a bot you could follow on each instance (like _@lastyear@mastodon.instance_) to playback your content from previous years as a private toot or DM.

**Comment by asos-simonj** [comment]
Very fair point on distinguising tragic events. 
On a related note, how does Mastodon handle people who pass away? What's the policy on accounts?
